source/slang/slang-ir.cpp

```diff
@@ -1757,6 +1765,25 @@ uint32_t _debugGetAndIncreaseInstCounter()
     {
 #if _WIN32 && defined(_MSC_VER)
         __debugbreak();
+#endif
+#ifdef SLANG_LINUX_FAMILY
```

Comment:
I think `SLANG_LINUX_FAMILY` is always defined? They are defined to `1` for linux platforms and `0` for non linux platforms so you should use `#if` instead of `#ifdef`.